packages/govuk-frontend/src/govuk/components/summary-list/template.njk

```diff
@@ -1,9 +1,12 @@
-{%- macro _actionLink(action) %}
+{%- macro _actionLink(action, cardTitle) %}
   <a class="govuk-link {%- if action.classes %} {{ action.classes }}{% endif %}" href="{{ action.href }}" {%- for attribute, value in action.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-    {{ action.html | safe if action.html else action.text }}
+    {{- action.html | safe if action.html else action.text -}}
     {%- if action.visuallyHiddenText -%}
       <span class="govuk-visually-hidden"> {{ action.visuallyHiddenText }}</span>
```

Comment:
To keep HTML file size down, should combine these two `govuk-visually-hidden` spans where possible?

```html
<a class="govuk-link" href="#">
  Delete
  
  <!-- Hidden text #1 -->
  <span class="govuk-visually-hidden"> name</span>
  
  <!-- Hidden text #2 -->
  <span class="govuk-visually-hidden"> (Undergraduate teaching assistant)</span>
</a>
```